packages/core/src/domain/configuration.ts

```diff
@@ -215,18 +226,44 @@ function getEndpoint(type: string, conf: TransportConfiguration, source?: string
   return `https://${host}/v1/input/${conf.clientToken}?${parameters}`
 }
 
+function getHost(type: EndpointType, conf: TransportConfiguration) {
+  if (NEW_INTAKE_DOMAIN_ALLOWED_SITES.indexOf(conf.site) !== -1) {
+    return `${type}.browser-intake-${conf.site}`
+  }
+  const oldTypes = {
+    [EndpointType.LOGS]: 'browser',
+    [EndpointType.RUM]: 'rum',
+    [EndpointType.TRACE]: 'public-trace',
+  }
+  return `${oldTypes[type]}-http-intake.logs.${conf.site}`
+}
+
+function getIntakeUrls(conf: TransportConfiguration) {
+  if (conf.proxyHost) {
+    return [`https://${conf.proxyHost}/v1/input/`]
+  }
+  const urls = [
+    `https://rum-http-intake.logs.${conf.site}/v1/input/`,
+    `https://browser-http-intake.logs.${conf.site}/v1/input/`,
+    `https://public-trace-http-intake.logs.${conf.site}/v1/input/`,
+  ]
+  if (NEW_INTAKE_DOMAIN_ALLOWED_SITES.indexOf(conf.site) !== -1) {
+    urls.push(
+      `https://rum.browser-intake-${conf.site}/v1/input/`,
+      `https://logs.browser-intake-${conf.site}/v1/input/`,
+      `https://trace.browser-intake-${conf.site}/v1/input/`
+    )
+  }
+  return urls
+}
+
 export function isIntakeRequest(url: string, configuration: Configuration) {
-  return (
-    getPathName(url).indexOf('/v1/input/') !== -1 &&
-    (haveSameOrigin(url, configuration.logsEndpoint) ||
-      haveSameOrigin(url, configuration.rumEndpoint) ||
-      haveSameOrigin(url, configuration.traceEndpoint) ||
-      (!!configuration.internalMonitoringEndpoint && haveSameOrigin(url, configuration.internalMonitoringEndpoint)) ||
-      (!!configuration.replica &&
-        (haveSameOrigin(url, configuration.replica.logsEndpoint) ||
-          haveSameOrigin(url, configuration.replica.rumEndpoint) ||
-          haveSameOrigin(url, configuration.replica.internalMonitoringEndpoint))))
-  )
+  for (const intakeUrl of configuration.intakeUrls) {
+    if (url.indexOf(intakeUrl) === 0) {
+      return true
+    }
+  }
+  return false
```

Comment:
Could be:
```suggestion
  return configuration.intakeUrls.some((intakeUrl) => url.indexOf(intakeUrl) === 0)
```

Reply:
Would the `.some()` work on IE 11? because I tried `.includes()` before and it didn't.

Reply:
Yes, IE9 supports `.some`: https://caniuse.com/?search=some

Reply:
Done